packages/settings-library/src/settings_library/redis.py

```diff
@@ -8,7 +8,7 @@
 from .basic_types import PortInt
 
 
-class RedisDatabase(int, Enum):
+class RedisDatabase(IntEnum):
```

Comment:
Anywhere else we might have the same issue?

Reply:
Nope, I've checked. this was the only one left